specification/ai/Azure.AI.Assistants/main.tsp

```diff
@@ -28,19 +28,26 @@ namespace Azure.AI {
     {
       type: OAuth2FlowType.implicit,
       authorizationUrl: "https://login.microsoftonline.com/common/oauth2/v2.0/authorize",
-      scopes: ["https://cognitiveservices.azure.com/.default"],
+      scopes: ["https://management.azure.com/.default"],
```

Comment:
Is this a temporary change, until the Agent service work to transfer it to the "AI Services" resource is done? As far as I know 1DP uses Entra ID with cognitiveservices.azure.com scope. If so, can you please keep the original line, comment it out and put a comment? Thanks!

Reply:
Fixed.